zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2987,6 +3036,41 @@ components:
           additionalProperties: true
           description: All the variables visible in the root scope.
           type: object
+    MigrateProcessInstanceRequest:
+      type: object
+      properties:
+        targetProcessDefinitionKey:
+          description: The key of process definition to migrate the process instance to.
+          type: integer
+          format: int64
+        mappingInstructions:
+          type: array
+          items:
+            $ref: "#/components/schemas/MigrateProcessInstanceMappingInstruction"
+        operationReference:
+          description: >
+            A reference key chosen by the user that will be part of all records resulting from this operation.
+            Must be > 0 if provided.
+          type: integer
+          format: int64
+          minimum: 1
+      required:
+        - targetProcessDefinitionKey
+        - mappingInstructions
+    MigrateProcessInstanceMappingInstruction:
+      type: object
+      description: |
+        The mapping instructions describe how to map elements from the source process definition to the target process definition.
+      properties:
+        sourceElementId:
+          description: The element id to migrate from.
```

```suggestion
          description: The element ID to migrate from.
```

🔧  I defer to a technical writer's review on this, but I think this is the way we capitalize ID. 

💭 It was also mentioned in [another PR](https://github.com/camunda/camunda/pull/22137#discussion_r1752288711) that [the guidance I wrote up](https://github.com/camunda/camunda/blob/main/docs/rest-controller.md#openapi-extension) suggested that id should not be capitalized. If ID should be capitalized, I will update that guidance to hopefully remove that confusion.

---

The [naming conventions of the C8 API guidelines](https://docs.google.com/document/d/1G9AmmNac-4QLGZ0LXQXa3FyeCrSJdIboaPt3-R6dWNw/edit#heading=h.8kew5enz4jtm) (based on guidelines created earlier by DevEx) suggest to use lowercase.

---

Naming conventions or code references aren't quite the same as uses in text; our fallback is the Google style guide, which also suggests capitalization. It's something currently under discussion, though, so there's no set internal standard either way.

---

To be consistent within the specification I will use uppercase for ID